setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = kaitaistruct
-version = attr: kaitaistruct.__version__
+version = attr: __version__
```

Comment:
What's the reason for this change? I haven't had any issues with the previous setting.

Reply:
I am planning to not to import, but extract this var with `ast.parse`. I remember I have done it for some projects, but I don't remember their names, probably I would have to recode that from scratch.

Reply:
Hm, not sure if this is worth the effort in our case. There are situations where `attr: module.__version__` doesn't work (see pypa/setuptools#1724, pypa/setuptools#1960), but `kaitaistruct` isn't affected by that.

If you really want to get `kaitaistruct.__version__` without importing the module, I would use the code from pypa/setuptools#1960 instead of reimplementing it from scratch.